db/migrations/00005_create_storage_diffs_table.sql

```diff
@@ -1,4 +1,12 @@
 -- +goose Up
+CREATE TYPE public.diff_status AS ENUM (
+    'new',
+    'transformed',
+    'unrecognized',
+    'noncanonical',
+    'unwatched'
```

Comment:
no real action item here but definitely want to think about how we can potentially accommodate multiple plugins/execute processes - where a diff that's `"unwatched"` for one plugin might still be relevant to another. Given that we're getting better about configuring which contracts we're tracking, I almost wonder if we can eventually discard this status entirely 🤔

Reply:
🤔  Yeah, this is a good call. Do you think it's worth removing this status now? We could leave those diffs with a status as 'new' for now? I think those diffs won't get constantly pulled if we've set `DiffBlocksFromHeadOfChain`.